### PR TITLE
new file:   packages/molenc/molenc.16.0.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/molenc/molenc.16.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  ["cp" "bin/molenc_standardize.py" "%{bin}%/molenc_standardize.py"]
+  ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+  ["cp" "bin/smi2png.py" "%{bin}%/molenc_smi2png.py"]
+  ["cp" "bin/molenc_smisur.py" "%{bin}%/molenc_smisur.py"]
+  ["cp" "bin/molenc_panascan.py" "%{bin}%/molenc_panascan.py"]
+  ["cp" "bin/molenc_scaffold.py" "%{bin}%/molenc_scaffold.py"]
+  ["cp" "bin/molenc_deepsmi.py" "%{bin}%/molenc_deepsmi.py"]
+]
+depends: [
+  "batteries"
```
Comment (kit-ty-kate) [inline packages/molenc/molenc.16.0.0/opam:26]:
```suggestion
  "batteries" {>= "3.0.0"}
```
This package is using `BatString.split_on_string`

File: packages/molenc/molenc.16.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  "bst"
+  "conf-graphviz"
+  "conf-python-3"
+  "conf-rdkit"
+  "cpm"
```
Comment (kit-ty-kate) [inline packages/molenc/molenc.16.0.0/opam:31]:
```suggestion
  "cpm" {>= "4.0.0"}
```
cpm < 4.0.0 did not have the module `Cpm`

File: packages/molenc/molenc.16.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  ["cp" "bin/molenc_type_atoms.py" "%{bin}%/molenc_type_atoms.py"]
+  ["cp" "bin/smi2png.py" "%{bin}%/molenc_smi2png.py"]
+  ["cp" "bin/molenc_smisur.py" "%{bin}%/molenc_smisur.py"]
+  ["cp" "bin/molenc_panascan.py" "%{bin}%/molenc_panascan.py"]
+  ["cp" "bin/molenc_scaffold.py" "%{bin}%/molenc_scaffold.py"]
+  ["cp" "bin/molenc_deepsmi.py" "%{bin}%/molenc_deepsmi.py"]
+]
+depends: [
+  "batteries"
+  "bst"
```
Comment (kit-ty-kate) [inline packages/molenc/molenc.16.0.0/opam:27]:
```suggestion
  "bst" {>= "2.0.0"}
```
Uses `Bst.Make` with only one parameter instead of two.

**Review by kit-ty-kate — CHANGES_REQUESTED** [review]
Could you improve the minimum requirements? Feel free to use our CI for that.

**Comment by UnixJunkie** [comment]
I approved all your version constraints, thanks.
What do you mean by minimum requirements? Is it something else?

**Comment by kit-ty-kate** [comment]
> I approved all your version constraints, thanks.
> What do you mean by minimum requirements? Is it something else?

I meant just looking out for further failures. I didn't test it entirely locally, I was only reading the CI failures but other failures were hidden by the previous ones.

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the additional fixes to the opam file upstream?

**Comment by UnixJunkie** [comment]
Yes, I will upstream the version constraints you added.
Thanks.

**Pull request merged**